tabs/system-setup/system-cleanup.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh -e
+
+. ../common-script.sh
+
+cleanup_system() {
+    printf "%b\n" "${YELLOW}Performing comprehensive system cleanup...${RC}"
+    case $PACKAGER in
+        apt-get|nala)
+            $ESCALATION_TOOL "$PACKAGER" clean
+            $ESCALATION_TOOL "$PACKAGER" autoremove -y
+            $ESCALATION_TOOL "$PACKAGER" autoclean
+            $ESCALATION_TOOL du -h /var/cache/apt
+            $ESCALATION_TOOL "$PACKAGER" clean
+            ;;
+        zypper)
+            $ESCALATION_TOOL "$PACKAGER" clean -a
+            $ESCALATION_TOOL "$PACKAGER" tidy
+            $ESCALATION_TOOL "$PACKAGER" cc -a
+            ;;
+        dnf)
+            $ESCALATION_TOOL "$PACKAGER" clean all
+            $ESCALATION_TOOL "$PACKAGER" autoremove -y
+            $ESCALATION_TOOL "$PACKAGER" remove "$(dnf repoquery --extras --exclude=kernel,kernel-\*)" -y
+            ;;
+        pacman)
+            $ESCALATION_TOOL "$PACKAGER" -Sc --noconfirm
+            $ESCALATION_TOOL "$PACKAGER" -Rns "$(pacman -Qtdq)" --noconfirm
+            if command -v paru >/dev/null 2>&1; then
+                paru -c
+            elif command -v yay >/dev/null 2>&1; then
+                yay -c
+            fi
```

Comment:
add `checkAURHelper` from common-script if you want to, if not use

```suggestion
            if command_exists paru >/dev/null 2>&1; then
                paru -c
            elif command_exists yay >/dev/null 2>&1; then
                yay -c
            fi
```